projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DockerRegistryService.java

```diff
@@ -45,9 +44,7 @@ public String builderImage() {
   }
 
   // TODO: Implement
-  public boolean dataJobImageExists(
-      String imageName, AWSCredentialsService.AWSCredentialsDTO awsCredentialsDTO) {
-    return registryType.equalsIgnoreCase("ecr")
-        && ecrRegistryInterface.checkEcrImageExists(imageName, awsCredentialsDTO);
+  public boolean dataJobImageExists(String imageName) {
+    return false;
```

Comment:
why this method return false?

Reply:
We extend the existing behavior to support Jfrog registry and ECR, but a generic Docker repository is still not supported.